lambda/health_package/metric-settings.json

```diff
@@ -1,4 +1,206 @@
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "IncomingRecords",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 0.99,
+    "Description": "The number of records successfully put to the Kinesis stream over the specified time period. This metric includes record counts from PutRecord and PutRecords operations. Minimum, Maximum, and Average statistics represent the records in a single put operation for the stream in the specified time period."
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.Success",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 0.99,
+    "Description": "Alert on 1% failure"
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.Records",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 50,
+    "Description": "The number of records retrieved from the shard, measured over the specified time period. Minimum, Maximum, and Average statistics represent the records in a single GetRecords operation for the stream in the specified time period."
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.Bytes",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 35000,
+    "Description": "The number of bytes retrieved from the Kinesis stream, measured over the specified time period. Minimum, Maximum, and Average statistics represent the bytes in a single GetRecords operation for the stream in the specified time period."
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.IteratorAgeMilliseconds",
+    "Statistic": "Maximum",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 5000,
+    "Maximum": 1800000,
+    "Description": "Maximum 1/2 an hour"
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "GetRecords.Latency",
+    "Statistic": "Average",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 5,
+    "Maximum": 26,
+    "Description": "The time taken (msecs) per GetRecords operation, measured over the specified time period."
+  },
+  {
+    "Namespace": "AWS/Kinesis",
+    "MetricName": "PutRecord.Success",
+    "Statistic": "Minimum",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 0.99,
```

Comment:
I think actually these accept 1% error metrics are probably not OK. We should probably set things like this to `Minimum: 1`, `EvaluationPeriods: 1` so the alarm goes off if we drop any events.